<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task1widget</class>
 <widget class="QWidget" name="task1widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>513</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Массивых N простых чисел:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask1_Output">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Время выполнения: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTask1_Time">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Введите N (количество простых чисел) от 1 до 10000:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxTask1_Input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTask1_Calculate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50;  
    color: white;
    border: none;
    padding: 10px;
    min-width: 10px;         
    min-height: 10px;          
    border-radius: 15px;      
}
QPushButton:hover {
    background-color: #45a049;  
}
QPushButton:pressed {
    background-color: #3e8e41; 
}</string>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTask1_Clean">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF9800;   
    color: white;
    border: none;
    padding: 10px;
    min-width: 10px;         
    min-height: 10px;          
    border-radius: 15px;     
}
QPushButton:hover {
    background-color: #F57C00;    
}
QPushButton:pressed {
    background-color: #E65100;    
}</string>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="buttonBack">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>36</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #f44336; 
    color: white;
    border: none;
    padding: 10px;
    min-width: 10px;
    min-height: 6px;
    border-radius: 15px;
}

QPushButton:hover {
    background-color: #d32f2f;  
}

QPushButton:pressed {
    background-color: #b71c1c;  
}</string>
     </property>
     <property name="text">
      <string>&lt;- Назад</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Задание 1. Массивых первых N простых чисел</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
